Ch 4

vtotient opened the pull request:
I also completed ticket #5 with this branch.

pritpalc commented on template-parts/title.php:
```diff
@@ -0,0 +1,7 @@
+<?php if( get_field('page_title') ): ?>
```
Did you create a field for the user to enable or disable the title? So for example you would do the check
```
<?php if( get_field('enable_page_title') ): ?>
<?php if( get_field('page_title') ): ?>
...
<?php endif; ?>
<?php endif; ?>

vtotient commented:
I separated the two tickets as requested. This PR now only contains issue #4. I added to the inc folder a file containing the wp generated php. I also added to the title.php file code for the enable check box. It now properly displays the title when the enable content box is clicked.
